cpp/roaring64map.hh

```diff
@@ -230,35 +271,70 @@ public:
         uint32_t end_high = highBytes(max);
         uint32_t end_low = lowBytes(max);
 
+        // We put std::numeric_limits<>::max in parentheses to avoid a
+        // clash with the Windows.h header under Windows.
+        const uint32_t maxUint32 = (std::numeric_limits<uint32_t>::max)();
```

Comment:
Nit: snake_case here and elsewhere :)

Reply:
I only found the one in this PR (and I have changed it as requested), but it certainly pops up in other PRs. Was there any other instance in this PR that I missed?